src/main/java/seedu/address/logic/parser/TimeslotRegex.java

```diff
@@ -0,0 +1,14 @@
+package seedu.address.logic.parser;
+
+/**
+ * Contains Regex Format for Date recognition in Raw User Inputs
+ */
+public class TimeslotRegex {
+
+    /* Timeslot Regex Format */
+    //=========== Date ================================================================================
+    public static final String DATE_SLASH_SHORT = "([0-9]{2})/([0-9]{2})/([0-9]{2})";
+    public static final String DATE_SLASH_LONG = "([0-9]{2})/([0-9]{2})/([0-9]{4})";
+    public static final String DATE_DASH_SHORT = "([0-9]{2})-([0-9]{2})-([0-9]{2})";
+    public static final String DATE_DASH_LONG = "([0-9]{2})-([0-9]{2})-([0-9]{4})";
```

Comment:
it would be better if this was more explicit to facilitate/improve resuability by someone who doesn't know regex, maybe something like `DD_SLASH_MM_SLASH_YY`, etc